### ISSUE TITLE
Move repo as member

### ISSUE THREAD

**Issue by lcnittl** [sphinx-contrib/github-administration#9]
Hi,

as member of sphinx-contrib, are we asked to ask permission before migrating a repo into the org?

In case yes, and (in case no, as I am writing this issue anyways already 😄), the repo of question is the following:

https://github.com/lcnittl/sphinxcontrib-scm

Cheers

**Comment by dhellmann** [comment]
If you're already a member of the org and admin of a repo then you should have the GitHub permissions to move a repo. You don't need to ask for permission from org administrators to do it, but you can use this repo to ask for help or advice, if you need it.

So, go right ahead and import whatever you like! :-)

**Comment by lcnittl** [comment]
Excellent – Thanks for clarifying!

I have just migrated it, and it seems that I have lost "admin" rights on the repo. Can you please give me those rights in [sphinxcontrib-scm](https://github.com/sphinx-contrib/scm) so I can access the repo's settings? Thanks in advance, and sorry for any inconvenience.

**Comment by lcnittl** [comment]
Is there a way for this that does not involve wasting your time?

Like creating a team (I can do that as member), and then giving that team admin access on transfer (there is an option to assign team access during transfer, but to be honest I did not check whether this was _just_ write permission, or if it was indeed admin).

**Comment by stephenfin** [comment]
> Excellent – Thanks for clarifying!
> 
> I have just migrated it, and it seems that I have lost "admin" rights on the repo. Can you please give me those rights in [sphinxcontrib-scm](https://github.com/sphinx-contrib/scm) so I can access the repo's settings? Thanks in advance, and sorry for any inconvenience.

Weird. I set it back to admin anyway so you should be good now. I guess we need to take a closer look at GitHub organisations and default permissions. Let's keep this issue open to track that.

**Comment by stephenfin** [comment]
Actually, looking at [this issue](https://github.community/t/permissions-after-transfer-into-organisation/2253/2), it seems this is a known issue. I guess we're blocked on that. Thankfully it's relatively painless to change permissions so asking an admin to do that is perfectly okay.

Given the above, I think we just need to document this particular side effect of migration and then we should close this issue.

**Comment by lcnittl** [comment]
Hmm, indeed. Thanks for digging that out!

It also seems that my idea with the teams would not work, as I do not believe _access_ means _admin_ (This is what pops up on transfer):

![image](https://user-images.githubusercontent.com/23134999/94051075-55f2e880-fdd7-11ea-8d1e-3f39edd79a1c.png)

True that, maybe an updated point 5 in the readme?

**Comment by Antoniomsopez** [comment]
/ # @
```coffeescript
[]() 
/
Duplicate of   @##
```

**Comment by Antoniomsopez** [comment]
<details><summary>Details</summary>
<p>

#42 

</p>
</details>
